frontend/src/components/App/Home/index.tsx

```diff
@@ -209,6 +209,17 @@ function useWarningSettingsPerCluster(clusterNames: string[]) {
   return warningLabels;
 }
 
+function getOrigin(cluster: Cluster) {
+  if (cluster.meta_data?.source === 'kubeconfig') {
+    return `Kubeconfig (${process.env.KUBECONFIG ?? '~/.kube/config'})`;
+  } else if (cluster.meta_data?.source === 'dynamic_cluster') {
+    return `Dynamic cluster`;
+  } else if (cluster.meta_data?.source === 'in_cluster') {
+    return 'Headlamp';
```

Comment:
It could be Headlamp is branded as something else, so we shouldn't use Headlamp here I think.

I think "In-Cluster" is maybe better?

Reply:
done